fedimint-bitcoind/src/lib.rs

```diff
@@ -66,17 +66,15 @@ dyn_newtype_define! {
 #[derive(Debug)]
 pub struct RetryClient<C> {
     inner: C,
-    max_retries: u16,
-    base_sleep: Duration,
+    sleep: Duration,
     task_handle: TaskHandle,
 }
 
 impl<C> RetryClient<C> {
     pub fn new(inner: C, task_handle: TaskHandle) -> Self {
         Self {
             inner,
-            max_retries: 10,
-            base_sleep: Duration::from_millis(10),
+            sleep: Duration::from_millis(100),
```

Comment:
We should still do some sort of back-off (exponential with an upper bound?).

Instead of failing we could just stop increasing the wait time after `max_retries`.